backend/printmach.ml

```diff
@@ -225,22 +225,25 @@ let operation' ?(print_reg = reg) op arg ppf res =
     let len = Array.length arg in
     fprintf ppf "csel %a ? %a : %a"
       (test tst) arg reg arg.(len-2) reg arg.(len-1)
-  | Ivalueofint -> fprintf ppf "valueofint %a" reg arg.(0)
-  | Iintofvalue -> fprintf ppf "intofvalue %a" reg arg.(0)
-  | Ivectorcast Bits128 ->
-    fprintf ppf "vec128->vec128 %a"
-    reg arg.(0)
-  | Iscalarcast Float32_as_float -> fprintf ppf "float32 as float %a" reg arg.(0)
-  | Iscalarcast (Float_of_int Float64) -> fprintf ppf "int->float %a" reg arg.(0)
-  | Iscalarcast (Float_to_int Float64) -> fprintf ppf "float->int %a" reg arg.(0)
-  | Iscalarcast (Float_of_int Float32) -> fprintf ppf "int->float32 %a" reg arg.(0)
-  | Iscalarcast (Float_to_int Float32) -> fprintf ppf "float32->int %a" reg arg.(0)
-  | Iscalarcast (Float_of_float32) -> fprintf ppf "float32->float %a" reg arg.(0)
-  | Iscalarcast (Float_to_float32) -> fprintf ppf "float->float32 %a" reg arg.(0)
-  | Iscalarcast (V128_of_scalar ty) ->
+  | Ireinterpret_cast Int_of_value -> fprintf ppf "value as int %a" reg arg.(0)
+  | Ireinterpret_cast Value_of_int -> fprintf ppf "int as value %a" reg arg.(0)
+  | Ireinterpret_cast Float32_of_float -> fprintf ppf "float as float32 %a" reg arg.(0)
+  | Ireinterpret_cast Float_of_float32 -> fprintf ppf "float32 as float %a" reg arg.(0)
+  | Ireinterpret_cast Int64_of_float -> fprintf ppf "int64 as float %a" reg arg.(0)
```

Comment:
It looks like there are inconsistencies
in the strings: for `x_of_y`, we sometimes
have `"x as y"` and sometimes `"y as x"`.

Reply:
Oops, fixed